### DISCUSSION TITLE
Approach for adding pkgtb firmware knowledge

### DISCUSSION THREAD

**Discussion by seud0nym** [hack-technicolor/hack-technicolor#255] [General]
Hi.

I have been working on methods for safely upgrading firmware and preserving root access on the VCNT-8 Technicolor CobraXh (Telstra Smart Modem Gen 3), which uses the pkgtb firmware file format (rather than rbi) and has a new U-boot based boot loader.

I am looking at starting with modifying the Safe Firmware Upgrade page, but I wanted to get your ideas on what approach should be taken. Should that page be modified to handle both firmware types, or do you want a separate page? My idea was just to have it all on the same page with links at the top to the different file formats.

**Comment by Whitewizarddonut** [comment]
You would need to get unlocked firmware. 
There is a programs which manages access called dms.

I am willing to trade access for monitoring firmware I can install on another person's router

**Reply by seud0nym** [reply]
I have unlocked firmware. I am asking about how the repository owners would like the documentation updated.

**Reply by LuKePicci** [reply]
I think we should collect info about this new firmware into a sparate section reflecting the same structure, mixing stuff about different platforms makes reading difficult. We had the same issues with homeware <=10

**Comment by Whitewizarddonut** [comment]
Firmware is updated by a policy on dms.There's locked versions and unlocked versions. I would just push the unlocked version and open admin accounts. 
But they report and updated are scheduled monthly.
It causes a issue where it says this site is untrustworthy on telstras pages and some others. 
What info do you want?
Sent from Yahoo Mail on Android 
 
  On Thu, 1 Sept 2022 at 8:08 am, Luca ***@***.***> wrote:   


I think we should collect info about this new firmware into a sparate section reflecting the same structure, mixing stuff about different platforms makes reading difficult. We had the same issues with homeware <=10

**Comment by seud0nym** [comment]
I have added a new Safe Firmware Upgrade page for pkgtb firmware. It is in Pull request #262, which also contains the repository updates and a couple of warnings on other relevant pages.

**Reply by LuKePicci** [reply]
Thanks, I hope to review it this weekend
